.github/workflows/ci.yaml

```diff
@@ -59,6 +59,26 @@ jobs:
         run: |
           make unit-tests
 
+  int-tests:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+          cache-dependency-path: "**/*.sum"
+      - name: Install kind
+        run: |
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.24.0/kind-linux-amd64
```

Comment:
can you add `kind` to list of requirements for dev env?

Reply:
done